src/main/webapp/src/app/app.component.html

```diff
@@ -1,21 +1,43 @@
-<!-- Template page: these components will appear on
-every page unless they are suppressed. -->
-<mat-toolbar id="growpod-top-bar" color="primary">
-  <mat-toolbar-row>
-    <a routerLink="/page/index"><span>GrowPod</span></a>
-    <span class="growpod-spacer"></span>
-    <a routerLink="/page/test-page1" routerLinkActive="active">
-      <span class="growpod-link">
-        Test Page 1
-      </span>
-    </a>
-    <a routerLink="/page/test-page2" routerLinkActive="active">
-      <span class="growpod-link">
-        Test Page 2
-      </span>
-    </a>
-  </mat-toolbar-row>
-</mat-toolbar>
-
-<!-- Routed page contents -->
-<router-outlet></router-outlet>
+<!-- Displays app contents if user is logged in, otherwise directs user to a home/login page -->
+<div *ngIf="loggedIn; else homePage">
+  <mat-toolbar id="growpod-top-bar" color="primary">
```

Comment:
We won't be using the id growpod-top-bar so this can be removed

Reply:
Addressed.